.azure-pipelines/build-template.yml

```diff
@@ -6,12 +6,16 @@ parameters:
   imageName: "khronosgroup/docker-images"
   tag:
   command: build
+  version:
 
 steps:
   - task: Docker@2
-    displayName: ${{ parameters.command }} ${{ parameters.tag }}
+    displayName: ${{ parameters.command }} ${{ parameters.tag }} ${{ parameters.version }}
     inputs:
       repository: ${{ parameters.imageName }}
       command: ${{ parameters.command }}
-      tags: ${{ parameters.tag }}
-      Dockerfile: Dockerfile.${{ parameters.tag }}
+      arguments: "--build-arg 'VERSION=${{ parameters.version }}'"
```

Comment:
Apparently this isn't going to work for the `buildAnPush`: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/docker?view=azure-devops#why-does-docker-task-ignore-arguments-passed-to-buildandpush-command

Should we separate that into a `build`+`push` or drop `push` altogether as there's a preference for manual, even for OpenXR?

Reply:
Done :)